src/spec/hooks/start.hook.js

```diff
@@ -1,4 +1,4 @@
-import chai from 'chai';
+import * as chai from 'chai';
 import chaiHttp from 'chai-http';
```

Comment:
chaiHttp provides a default import and doesn't need `* as chaiHttp` to properly import. This is different from the necessary change to `import * as chai from 'chai'` because chai v5 doesn't provide a default import